browsergym/core/src/browsergym/core/action/highlevel.py

```diff
@@ -514,6 +514,56 @@ def to_python_code(self, action):
         # return the constructed python code
         return python_code
 
+    def to_tool_description(self, api="openai") -> list[dict]:
```

Comment:
Seems like a good candidate for a unit test. But perhaps we could isolate it into a separate function first. Perhaps `tool = get_tool(tool_name, action)`.